examples/contour/01-crds.yaml

```diff
@@ -328,6 +328,11 @@ spec:
                       required:
                       - caSecret
                       type: object
+                    enableFallbackCertificate:
```

Comment:
is this v1beta1.ingressRoutes changing expected? only httpProxy cr right?

Reply:
oh no, that's not expected! Must be something with the generators.

Reply:
Some of the IngressRoute stuff uses structs shared from HTTPProxy; is this the case here?

Reply:
I opened https://github.com/projectcontour/contour/pull/2511 to resolve.

Reply:
👍